package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "start": "webpack-dev-server",
     "dev": "REDASH_BACKEND=https://dev.redashapp.com npm start",
+    "bundle": "bin/bundle-extensions",
```

Comment:
Is there a reason to keep bundle as a separate task? Maybe worth to make sure it's always invoked when running build/watch/start?

Reply:
I think this goes back to my explanation for why I added the Makefile. I tried looking into using `prebuild`, `prewatch` etc. here to always invoke bundle first. However, this would only work in the container (so it would be fine in the `Dockerfile`) but attempting to run `npm run build` locally wouldn’t work because the bundle script doesn’t have access to the extensions dependency.

For now, perhaps the `Makefile` can have a couple of more commands for `watch` and `start`. In the future, we might find a way such that the build steps in the container are done in a separate directory than the volume mounting so that this is no longer an issue.